store/rootmulti/store.go

```diff
@@ -576,6 +576,73 @@ func (rs *Store) pruneStores() error {
 	return nil
 }
 
+// PruneHistoryVersions prunes the history versions of each substore based on
+// the pruning options.
+func (rs *Store) PruneHistoryVersions() error {
+	pruningOptions := rs.GetPruning()
+	latestVersion := getLatestVersion(rs.db)
+
+	if pruningOptions.Interval == 0 {
+		rs.logger.Info("pruning returns with no op because pruning interval is set to 0\n")
```

Comment:
Can we remove the log?

Reply:
removed `PruneHistoryVersions` and make use of `PruneStores`